src/pages/login/CertificateAdd.tsx

```diff
@@ -1,16 +1,24 @@
 import type { FC } from "react";
-import { Col, Notification, Row, useNotify } from "@canonical/react-components";
+import {
+  CodeSnippet,
+  Col,
+  Notification,
+  Row,
+  useNotify,
+} from "@canonical/react-components";
 import { Navigate } from "react-router-dom";
 import { useAuth } from "context/auth";
 import Loader from "components/Loader";
 import CertificateAddForm from "pages/login/CertificateAddForm";
 import NotificationRow from "components/NotificationRow";
 import CustomLayout from "components/CustomLayout";
-import HelpLink from "components/HelpLink";
+import { useSettings } from "context/useSettings";
 
 const CertificateAdd: FC = () => {
   const { isAuthenticated, isAuthLoading } = useAuth();
   const notify = useNotify();
+  const { data: settings } = useSettings();
+  const hasCertificate = !settings?.client_certificate;
```

Comment:
```suggestion
  const hasCertificate = settings?.client_certificate;
```
This should not be inverted, I think?

Reply:
Yes, sorry, I believe I left this in by accident after testing.